packages/website/src/components/Announcement.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import Text from './Text';
+import { H4 } from './Heading';
+import Box from './Box';
+import Section from './Section';
+
+interface AnnouncementProps {
+  location: Object;
+}
+
+const AnnouncementBox = styled(Box)`
+  background: #ffe9bf;
+  width: 60%;
+  padding: ${({ theme }) => theme.space.medium} ${({ theme }) => theme.space.large};
+  margin-bottom: ${({ theme }) => theme.space.medium};
+  border-radius: 4px;
+`;
+
+const redirected = (props): boolean => {
+  const params = new URLSearchParams(props.search);
+  const from_old_website: string = params.get('from-old-website')!;
+  return from_old_website == 'true' ? true : false;
+};
+
+const from_where = props => {
+  const params = new URLSearchParams(props.search);
+  const from_which_page: string = params.get('archived-page')!;
+  return (
+    <Text>
+      You can still find the page you were trying to visit in our archive:{' '}
+      <a href={from_which_page}>{from_which_page}</a>
```

Comment:
Link do have a special style, so you have to use something like:

```
const AnnouncementLink = styled.a`
  border-bottom: 1px dotted ${({ theme }) => theme.colors.bodyCopy};
`;
```

Reply:
oh okay 👌